solr/solrj/src/test/org/apache/solr/client/solrj/TestLBHttp2SolrClient.java

```diff
@@ -156,7 +158,7 @@ public void testSimple() throws Exception {
       // Start the killed server once again
       solr[1].startJetty();
       // Wait for the alive check to complete
-      Thread.sleep(1200);
+      Thread.sleep(1200 * 5);
```

Comment:
1.2 seconds times five?

Reply:
I have no memory of this 🤔 - will remove.